### ISSUE TITLE
Still maintaining this code?

### ISSUE THREAD

**Issue by Ircama** [leonsoft-kras/transmisson-remote-gui#877]
Still maintaining transmisson-remote-gui?
Asking this as I have some updates on https://github.com/Ircama/transmisson-remote-gui that I might consider to merge with a maintained version.
Let me know, thanks
Regards
Ircama

**Comment by leonsoft-kras** [comment]
Hi!
I'm not against. tell what improvements will be implemented?

**Comment by Ircama** [comment]
Hi,
Thanks!
I made small changes to the translation strings and to some user interface option in relation to translation aspects.
I originally based my updates on xorkrus/transmisson-remote-gui, which has an old synapse library (6 years ago) and everything works. Your updates are anyway more recent than xorkrus' ones and I am considering to also push my work to your repository. I have now forked your code and recompiled it locally (I also followed changes reported in http://forum.lazarus.freepascal.org/index.php?topic=29034.0 for the json library, to avoid conflicts with the new on included in Lazarus). I anyway get an error when executing it ("The ordinal 3019 could not be located in the dynamic link library LIBEAY32.dll"). By replacing your synapse\source\lib (modified 8 months ago) with xorkrus' one, the error popup does not appear any more. Did you make some change to the synapse library? I see you updated it 8 months ago, is there a reason? Conversely, might I replace yours with the original one in http://sourceforge.net/projects/transgui/files/5.0.1/transgui-5.0.1-src.zip (which should be hopefully the same as xorkrus')?

**Comment by leonsoft-kras** [comment]
Hi!
I used the basic source of the author, version 5.01.
I made a lot of changes and added various functional (on the proposals in the issues). you can compare my and your sources, for example using Total Commander.
I think updated synapse. I went to the site and downloaded the new version. any problems with the replacement work and did not notice.

I saw the move зкщоусе to http://sourceforge.net/projects/transgui.
apparently the author of the project is found. I did not compare what he did. If there is 5.01, then I was initially version 5.01

I did not understand what version of the project are invited to see and unite. Take your version? 
I briefly looked - almost every file between your and my version has differences (not small).

**Comment by Ircama** [comment]
Ok, I guess you removed the original synapse directory and replaced it with the one available here: http://www.ararat.cz/synapse/doku.php/download
So I tested http://www.ararat.cz/synapse/lib/exe/fetch.php/file:synapse40.zip and verified the same error. I then installed the latest Win32 OpenSSL from https://slproweb.com/products/Win32OpenSSL.html and I have no more this issue. Did you perform the same?
